src/lib.rs

```diff
@@ -846,7 +846,9 @@ impl Builder {
 
     /// Add an argument to be passed straight through to clang.
     pub fn clang_arg<T: Into<String>>(mut self, arg: T) -> Builder {
-        self.options.clang_args.push(arg.into());
+        for arg in arg.into().split_whitespace() {
+            self.options.clang_args.push(arg.to_string());
```

Comment:
It's not clear to me this is the right fix...

This would break working-code for example that could pass:

```
.clang_arg("-I")
.clang_arg("/Directory/with spaces/")
```

`clang_arg` is meant to support a single argument passed to clang.